articles/migrations/index.md

```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
 
 If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
-### Introducing Resource Owner Support for oauth/token Endpoint
+---
+
+## 2018 Upcoming Endpoint Migrations
```

Comment:
From the discussions I gathered that these migrations will start within 2018 (grace period start) and finish within 2018 (mandatory opt-in). Maybe the header should be `Upcoming Endpoint Migrations` and then clarify in the introductory text that the migrations will start within 2018 (as soon as we publish migration guides) and end (mandatory opt in) within 2019.

As is now I understand that these migrations will start and end within 2018.